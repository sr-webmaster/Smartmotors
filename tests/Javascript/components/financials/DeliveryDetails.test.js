import { mount } from '@vue/test-utils'
import Component from '~/components/financials/DeliveryDetails.vue'

describe('DeliveryDetails', () => {
  describe('Snapshot', () => {
    test('default', async () => {
      const wrapper = mount(Component)
      await wrapper.vm.$nextTick()
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
