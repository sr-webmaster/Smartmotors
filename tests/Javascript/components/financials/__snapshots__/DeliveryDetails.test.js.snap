// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeliveryDetails Snapshot default 1`] = `
<div
  class="v-card v-sheet theme--light"
>
  <div
    class="v-card__text"
  >
    <h6>
      Date Delivered:
    </h6>
     
    <h1>
      9/15/2019
    </h1>
     
    <h6>
      Est. Date Delivered:
    </h6>
     
    <h1>
      9/14/2019
    </h1>
     
    <h6>
      Date Written:
    </h6>
     
    <h1>
      9/1/2019
    </h1>
  </div>
</div>
`;
